index.html

```diff
@@ -2,98 +2,226 @@
+                    sign-ups required.
+                </p>
+                <ul class="categories hex">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                </ul>
+                <div class="see-project">
+                    <a href="#"><button type="button" class="hex">See Project</button></a>
+                </div>
+            </div>
+        </section>
+        <!-- project 3 -->
+        <section class="grid-items other-projects">
+            <img class="project-img" src="images/projects/project-3.png" alt="Project-1" />
+            <div class="details">
+                <div class="titles">
+                    <h3 class="project-title">Tonic</h3>
+                    <ul class="canopy">
+                        <li>CANOPY</li>
+                        <li class="bullets"><span class="dot"></span></li>
+                        <li class="o-li">Back End Dev</li>
+                        <li class="bullets"><span class="dot"></span></li>
+                        <li class="o-li">2015</li>
+                    </ul>
+                </div>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or
+                    sign-ups required.
+                </p>
+                <ul class="categories hex">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                </ul>
+                <div class="see-project">
+                    <a href="#"><button type="button" class="hex">See Project</button></a>
+                </div>
+            </div>
+        </section>
+        <section class="grid-items other-projects">
+            <img class="project-img" src="images/projects/project-4.png" alt="Project-1" />
+            <div class="details">
+                <div class="titles">
+                    <h3 class="project-title">Multi-Post Stories</h3>
+                    <ul class="canopy">
+                        <li>CANOPY</li>
+                        <li class="bullets"><span class="dot"></span></li>
+                        <li class="o-li">Back End Dev</li>
+                        <li class="bullets"><span class="dot"></span></li>
+                        <li class="o-li">2015</li>
+                    </ul>
+                </div>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or
+                    sign-ups required.
+                </p>
+                <ul class="categories hex">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                </ul>
+                <div class="see-project">
+                    <a href="#"><button type="button" class="hex">See Project</button></a>
+                </div>
+            </div>
+        </section>
+    </div>
```

Comment:
- [ ] Since the works section is a standalone part of the project, kindly consider using the section tag for it instead of the div tag. 
- [ ] However, you don't need to use the section tag for the individual project cards container. Kindly consider changing those ones to a div or article tag instead of the section tags